### ISSUE TITLE
can’t set CodeLLDB library path

### ISSUE THREAD

**Issue by kelvin13** [swift-server/vscode-swift#365]
**Describe the bug**

the following error appears on application startup

```
Failed to setup CodeLLDB for debugging of Swift code. 
Debugging may produce unexpected results. 
Error: "LLDB failed to provide a library path"
```

**Expected behavior**

i tried inputting a path to `liblldb.so` into the CodeLLDB library path setting, but i still get the same error.

Environment**
 - OS: ubuntu 20.04
 - Swift version: 7/6/22 nightly
 ```
Swift version 5.8-dev (LLVM 396c6969ee90ef4, Swift 02efc3a2d28e348)
Target: x86_64-unknown-linux-gnu
```
 - Visual Studio Code version: 1.69.1
 - vscode-swift version: 0.6.0

**Additional context**
additional discussion here: https://forums.swift.org/t/vscode-can-t-set-codelldb-library-path/59024

**Comment by adam-fowler** [comment]
I forgot we added a workaround to the python scripts not working. So not sure why you are still seeing issues. Can you go into the settings, set Swift->Advanced->Diagnostics to true and then restart the extension either by restarting VSCode or running command `Developer: reload window`. Open a Swift project and then open the Swift Output pane. Can you include the contents of the window here?

Also one other question, is swift in your path, or are you setting the swift path in the extension settings

**Comment by kelvin13** [comment]
i got:

```
14:16:17: Swift version 5.8-dev (LLVM 396c6969ee90ef4, Swift 02efc3a2d28e348)
14:16:17: Swift Path: /home/klossy/.swiftenv/shims
14:16:17: Toolchain Path: /home
14:16:17: Runtime Library Path: /home/klossy/.swiftenv/versions/DEVELOPMENT-SNAPSHOT-2022-07-06-a/usr/lib/swift/linux
14:16:17: Default Target: x86_64-unknown-linux-gnu
14:16:18: Failed to setup CodeLLDB: Error: "LLDB failed to provide a library path"
```

swift is in my path; it was put there by swiftenv:

```
/home/klossy/.swiftenv/bin:/home/klossy/.swiftenv/shims
```

**Comment by adam-fowler** [comment]
Ah I think I might know what it is. I believe swiftenv uses a symbolic link to point to the correct version of swift if I remember right. We may not be dealing with that correctly. 

As an aside what is the output when you type `lldb --version` in a terminal.

**Comment by adam-fowler** [comment]
Temporarily you could probably get around this by setting the `swift.path` setting in vscode to the swift bin folder `/home/klossy/.swiftenv/versions/DEVELOPMENT-SNAPSHOT-2022-07-06-a/usr/bin`

**Comment by kelvin13** [comment]
> Temporarily you could probably get around this by setting the `swift.path` setting in vscode to the swift bin folder `/home/klossy/.swiftenv/versions/DEVELOPMENT-SNAPSHOT-2022-07-06-a/usr/bin`

this fixed the error for me! but it’s probably not an ideal workaround, because this path needs to be updated every time i switch swiftenv toolchains, which i do quite frequently as a package author. 


```
$ lldb --version
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib/python3/dist-packages/lldb/__init__.py", line 11409, in <module>
    class SBTraceOptions(object):
  File "/usr/lib/python3/dist-packages/lldb/__init__.py", line 11468, in SBTraceOptions
    __swig_destroy__ = _lldb.delete_SBTraceOptions
AttributeError: module '_lldb' has no attribute 'delete_SBTraceOptions'
lldb version 13.0.0 (https://github.com/apple/llvm-project.git revision 396c6969ee90ef48daaa8fe98d9b24c6a7fc4c98)
Swift version 5.8-dev (LLVM 396c6969ee90ef4, Swift 02efc3a2d28e348)
```

**Comment by adam-fowler** [comment]
@kelvin13 I had a further look into this. When you run swift using the $PATH setup by `swiftenv` you aren't running swift directly. You are running a shim which is a shell script which finds the version of swift you want to run and then runs it. This is a kind of non-standard way of running swift. Although it has become a standard on linux given how many people are using `swiftenv`. I don't have an immediate solution to this issue, apart from maybe use [`swiftbox`](https://github.com/stevapple/swiftbox) instead.

I guess I could recognise when someone has installed swift with swiftenv and use another method for finding `lldb`.

**Comment by kelvin13** [comment]
yes please, `swiftenv` for a lot of people *is* the standard way of using swift, especially since it also works on macOS through homebrew. 

based on its README `swiftbox` only supports ubuntu and amazon linux, and given how deeply entrenched `swiftenv` is in my workflows migrating to `swiftbox` is a non-starter.

**Comment by adam-fowler** [comment]
Fixed by #396
